### ISSUE TITLE
Messages automatically marked as read

### ISSUE THREAD

**Issue by vitalyster** [EionRobb/skype4pidgin#612]
I checked code and it looks the same as other plugins where messages marked as read by signal. Maybe Skype Web API marks messages on fetch?

**Comment by EionRobb** [comment]
Are you trying to say it's a bug to mark messages as 'read'?  Or is this a feature request?

**Comment by vitalyster** [comment]
Yes, looks like a bug - it should not mark as read until "conversation-updated" received, but it marks

**Comment by vitalyster** [comment]
I did update the issue, there is no read marks just cleared notifications
